### DISCUSSION TITLE
Bug with Blazor was 8 with authentication

### DISCUSSION THREAD

**Discussion by mostefaiamine** [IvanJosipovic/BlazorApplicationInsights#258] [General]
AI is working well when I follow the documentation for simple WASM apps. But when I try to plug it in my app (WASM 8, authentication), I get this error:
icrosoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: Could not find 'appInsights.updateCfg' ('updateCfg' was undefined).
      Error: Could not find 'appInsights.updateCfg' ('updateCfg' was undefined)

**Comment by IvanJosipovic** [comment]
Please double check that you followed all the steps, https://github.com/IvanJosipovic/BlazorApplicationInsights?tab=readme-ov-file#install-on-blazor-webassembly-standalone-app
